eng/Versions.props

```diff
@@ -48,19 +48,19 @@
     <BasicUndoVersion>0.9.3</BasicUndoVersion>
     <BenchmarkDotNetVersion>0.12.1</BenchmarkDotNetVersion>
     <DiffPlexVersion>1.4.4</DiffPlexVersion>
-    <EnvDTEVersion>8.0.2</EnvDTEVersion>
-    <EnvDTE80Version>8.0.0</EnvDTE80Version>
+    <EnvDTEVersion>16.9.30921.310</EnvDTEVersion>
+    <EnvDTE80Version>16.9.30921.310</EnvDTE80Version>
     <FakeSignVersion>0.9.2</FakeSignVersion>
     <HumanizerCoreVersion>2.2.0</HumanizerCoreVersion>
     <ICSharpCodeDecompilerVersion>6.1.0.5902</ICSharpCodeDecompilerVersion>
-    <MicrosoftBuildVersion>15.3.409</MicrosoftBuildVersion>
-    <MicrosoftBuildFrameworkVersion>15.3.409</MicrosoftBuildFrameworkVersion>
+    <MicrosoftBuildVersion>16.5.0</MicrosoftBuildVersion>
+    <MicrosoftBuildFrameworkVersion>16.5.0</MicrosoftBuildFrameworkVersion>
     <MicrosoftBuildLocatorVersion>1.2.6</MicrosoftBuildLocatorVersion>
-    <MicrosoftBuildRuntimeVersion>15.3.409</MicrosoftBuildRuntimeVersion>
-    <MicrosoftBuildTasksCoreVersion>15.3.409</MicrosoftBuildTasksCoreVersion>
+    <MicrosoftBuildRuntimeVersion>16.5.0</MicrosoftBuildRuntimeVersion>
+    <MicrosoftBuildTasksCoreVersion>16.5.0</MicrosoftBuildTasksCoreVersion>
```

Comment:
Should we introduce a `MicrosoftBuildPackagesVersion`?

Reply:
Done.